jgt_common/assert_.py

```diff
@@ -14,6 +15,16 @@ def _msg_concat(prefix, body):
     return "{} {}".format(prefix, body) if prefix else body
 
 
+def _percent_diff(a, b, precision=2):
```

Comment:
I think it'd be worth throwing this in `__init__.py` and adding a few tests for it.

Reply:
I'm cool with that. Will make that change.

Reply:
Pay no attention to that imposter! (Though I agree with his point)

Reply:
Bump.